node/src/reactor/participating/tests.rs

```diff
@@ -301,6 +301,16 @@ async fn run_equivocator_network() {
         }
     }
 
+    assert!(
+        !switch_blocks
+            .last()
+            .expect("missing switch block")
+            .next_era_validator_weights()
+            .expect("missing validator weights")
+            .contains_key(&alice_pk),
+        "Alice should have been evicted."
```

Comment:
Shouldn't that be the other way around? If we expect Alice to be evicted then she shouldn't be in `next_era_validator_weights` which means we should fail here if she is - we don't need the negation.

Reply:
NVM. I was wrong.